clang/include/clang/Sema/Sema.h

```diff
@@ -2667,6 +2675,10 @@ class Sema final {
   QualType BuiltinChangeSignedness(QualType BaseType, UTTKind UKind,
                                    SourceLocation Loc);
 
+  QualType BuildCountAttributedArrayType(
+      QualType WrappedTy, Expr *CountExpr,
+      const llvm::SmallVector<TypeCoupledDeclRefInfo, 1> &Decls);
```

Comment:
Would it be better to use an `ArrayRef<TypeCoupledDeclRefInfo>` here? If not, I think the way to pass in an LLVM container is something like this:


```c++
const llvm::SmallVectorImpl<TypeCoupledDeclRefInfo> &Decls);
```